src_c/mixer.c

```diff
@@ -842,6 +908,7 @@ PyMethodDef sound_methods[] = {
     {"get_volume", snd_get_volume, METH_NOARGS, DOC_MIXER_SOUND_GETVOLUME},
     {"get_length", snd_get_length, METH_NOARGS, DOC_MIXER_SOUND_GETLENGTH},
     {"get_raw", snd_get_raw, METH_NOARGS, DOC_MIXER_SOUND_GETRAW},
+    {"copy", snd_copy, METH_NOARGS, DOC_MIXER_SOUND_COPY},
```

Comment:
To be coherent with the rest of the API, the same `snd_copy` c function should be used for the `__copy__` method. Both methods should also be added to the stubs (both extended, `__copy__` doesn't need to be in the docs).
```suggestion
    {"copy", snd_copy, METH_NOARGS, DOC_MIXER_SOUND_COPY}, {"__copy__", snd_copy, METH_NOARGS, DOC_MIXER_SOUND_COPY},
```